tests/text/test_bleu.py

```diff
@@ -27,12 +27,12 @@
 smooth_func = SmoothingFunction().method2
 
 
-def _compute_bleu_metric_nltk(list_of_references, hypotheses, weights, smoothing_function, **kwargs):
-    hypotheses_ = [hypothesis.split() for hypothesis in hypotheses]
-    list_of_references_ = [[line.split() for line in ref] for ref in list_of_references]
+def _compute_bleu_metric_nltk(predictions, list_of_targets, weights, smoothing_function, **kwargs):
```

Comment:
```suggestion
def _compute_bleu_metric_nltk(preds, targets, weights, smoothing_function, **kwargs):
```

Reply:
Ok. I will update by tomorrow/day-after.

Reply:
@Borda @stancld: The requirements of the issue are being changed every iteration. Let us finalize one. If `preds` and `targets` are final, please update the requirements of issue #686. I will modify those 4 metrics, accordingly.

Reply:
@ashutoshml It looks like the whole `text` package has a different naming than the rest of `tm` package, so this was likely led to our confusion.

cc: @Borda (shouldn't we have `preds, target`, without `s` in `targets` as it is the case in other packages within `tm`?

Reply:
my typo, primary is consistency across all TM... :)